src/kakarot/execution_context.cairo

```diff
@@ -334,11 +334,8 @@ namespace ExecutionContext {
     func jump{range_check_ptr}(
         self: model.ExecutionContext*, new_pc_offset: felt
     ) -> model.ExecutionContext* {
-        alloc_locals;
-
-        let is_nn_pc = is_nn(new_pc_offset);
-        let is_le_bytecode_len = is_le(new_pc_offset, self.call_context.bytecode_len - 1);
-        if (is_nn_pc + is_le_bytecode_len != 2) {
+        let out_of_range = is_le(self.call_context.bytecode_len, new_pc_offset);
```

Comment:
will this return true if new_pc_offset > range check?

Reply:
actually jump arg is taken from the stack, see 

```
        let (stack, offset) = Stack.pop(ctx.stack);
        let ctx = ExecutionContext.update_stack(ctx, stack);
        let ctx = ExecutionContext.jump(ctx, offset.low);
```

so currently there is no chance pc update be >= 2 ** 128.

In the coming PRs, I'm adding checks for the stack output and consequently a check in `exec_jump` that `high=0`.